app.py

```diff
@@ -1,12 +1,9 @@
-from flask import Flask, render_template, redirect, url_for, request, session, flash, jsonify
-from urllib.request import urlopen
-from bs4 import BeautifulSoup
-import requests
-import csv
-import json
-import os
+from flask import Flask, render_template, redirect, url_for, request, flash, jsonify
+from scraping import scrape_website, save_to_csv, clear_csv_file
+from database import store_analysis_results_in_database
 
 app = Flask(__name__)
+app.secret_key = 'your_secret_key'  # Setting a secret key for flash messages
```

Comment:
What is this secret for?

Reply:
> What is this secret for?
Greetings @stephane-segning 
Please Sir, concerning the above secret key, I've been trying to figure it out.
Wish to inform you that I and my team did not add any key to the app.py

Reply:
Then remove it na